firebase.go

```diff
@@ -33,7 +33,7 @@ type FirebaseManifest struct {
 
 func (mf FirebaseManifest) processRedirects(w http.ResponseWriter, r *http.Request) (bool, error) {
 	for _, redirect := range mf.Redirects {
-		pattern, err := CompileExtGlob(redirect.Source)
+		pattern, err := CompileExtGlob("/" + strings.TrimPrefix(redirect.Source, "/"))
```

Comment:
To be compatible with Firebase, patterns should start with a `/`, and be matched against `r.URL.Path` (which also starts with a `/`).